.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
```

Comment:
For some reason I can't suggest code on this one, but could we also add this one in:

```yml
  - name: Tauri Bug Report
    url: https://github.com/tauri-apps/tauri/issues/new/choose
    about: Found a bug in Tauri? Let us know over on the Tauri repo
```